src/main/java/com/alevelhw/module2/exception/IncorrectReadStringException.java

```diff
@@ -0,0 +1,16 @@
+package com.alevelhw.module2.exception;
+
+public class IncorrectReadStringException extends Exception {
+    private final String detail;
+
+    public IncorrectReadStringException(String detail) {
+        this.detail = detail;
+    }
```

Comment:
There is a standard constructor for this
`public IncorrectReadStringException(String message) {
        super(message);
    }`